yaml/xpu_functions.yaml

```diff
@@ -361,6 +361,12 @@ supported:
   - hardsigmoid_
   - hardsigmoid_backward.grad_input
   - hardsigmoid_backward
+  - log_sigmoid.out
+  - log_sigmoid
```

Comment:
no need to register log_sigmoid.out/log_sigmoid, It's not structure delegation, CUDA doesn't register too

Reply:
Removed.